### PR TITLE
Update CMakeList.txt

### PR DESCRIPTION
**Pull request by bovirus**
@Shatur 

Pelase check and merge.
Please try to make a new installer.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -33,6 +37,19 @@ set(CPACK_NSIS_DISPLAY_NAME ${APPLICATION_NAME})
...
+   VIProductVersion \"${VERSION}\"
+   VIFileVersion    \"${VERSION}.0\"
+   VIAddVersionKey  \"ProductName\" \"${APPLICATION_NAME}\"
+   VIAddVersionKey  \"ProductVersion\" \"${VERSION}\"
+   VIAddVersionKey  \"CompanyName\" \"${COMPANY_NAME}\"
+   VIAddVersionKey  \"LegalCopyright\" \"${LEGAL_COPYRIGHT}\"
+   VIAddVersionKey  \"FileDescription\" \"${APPLICATION_NAME} Installer\"
+   VIAddVersionKey  \"FileVersion\" \"${VERSION}\"
+   VIAddVersionKey \"InternalName\" \"${APPLICATION_NAME}\"
+)
```
Comment (Shatur) [inline CMakeLists.txt:52]: Could you adjust formatting?
```suggestion
set(CPACK_NSIS_EXTRA_DEFINES 
   VIProductVersion \"${VERSION}\"
   VIFileVersion \"${VERSION}.0\"
   VIAddVersionKey \"ProductName\" \"${APPLICATION_NAME}\"
   VIAddVersionKey \"ProductVersion\" \"${VERSION}\"
   VIAddVersionKey \"CompanyName\" \"${COMPANY_NAME}\"
   VIAddVersionKey \"LegalCopyright\" \"${LEGAL_COPYRIGHT}\"
   VIAddVersionKey \"FileDescription\" \"${APPLICATION_NAME} Installer\"
   VIAddVersionKey \"FileVersion\" \"${VERSION}\"
   VIAddVersionKey \"InternalName\" \"${APPLICATION_NAME}\"
)
```

**Review by Shatur — CHANGES_REQUESTED** [review]
Thanks! Here is a few suggestions:

**Comment by Shatur** [comment]
>  Please try to make a new installer.

I currently don't have access to Windows, have you checked it yourself?

File: CMakeLists.txt [hunk 2]
```diff
@@ -8,6 +8,10 @@ project(crow-translate
 )
 
 set(APPLICATION_NAME "Crow Translate")
+set(COMPANY_NAME "Crow Translate")
```
Comment (Shatur) [inline CMakeLists.txt:11]: Please just duplicate `APPLICATION_NAME` since there is not company.

File: CMakeLists.txt [hunk 3]
```diff
@@ -33,6 +37,19 @@ set(CPACK_NSIS_DISPLAY_NAME ${APPLICATION_NAME})
 set(CPACK_NSIS_INSTALLED_ICON_NAME ${EXECUTABLE_NAME}.exe)
 set(CPACK_NSIS_EXECUTABLES_DIRECTORY .)
 set(CPACK_NSIS_MUI_FINISHPAGE_RUN ${EXECUTABLE_NAME}.exe)
+
+# Setup installer exe properties
+set (CPACK_NSIS_EXTRA_DEFINES 
+   VIProductVersion \"${VERSION}\"
+   VIFileVersion    \"${VERSION}.0\"
```
Comment (Shatur) [inline CMakeLists.txt:44]: I believe it should be like this:
```suggestion
   VIProductVersion \"${CMAKE_PROJECT_VERSION}\"
   VIFileVersion    \"${CMAKE_PROJECT_VERSION}.0\"
```

File: CMakeLists.txt [hunk 4]
```diff
@@ -33,6 +37,19 @@ set(CPACK_NSIS_DISPLAY_NAME ${APPLICATION_NAME})
...
+   VIAddVersionKey  \"ProductName\" \"${APPLICATION_NAME}\"
+   VIAddVersionKey  \"ProductVersion\" \"${VERSION}\"
+   VIAddVersionKey  \"CompanyName\" \"${COMPANY_NAME}\"
+   VIAddVersionKey  \"LegalCopyright\" \"${LEGAL_COPYRIGHT}\"
+   VIAddVersionKey  \"FileDescription\" \"${APPLICATION_NAME} Installer\"
+   VIAddVersionKey  \"FileVersion\" \"${VERSION}\"
```
Comment (Shatur) [inline CMakeLists.txt:50]:
```suggestion
   VIAddVersionKey  \"FileVersion\" \"${CMAKE_PROJECT_VERSION}\"
```

Comment (bovirus) [reply CMakeLists.txt:50]: OK. Thanks.

File: CMakeLists.txt (hunk 2 above)
Comment (bovirus) [reply CMakeLists.txt:11]: OK: Pelase change it.

File: CMakeLists.txt (hunk 3 above)
Comment (bovirus) [reply CMakeLists.txt:44]: OK. Thanks.

File: CMakeLists.txt (hunk 4 above)
Comment (bovirus) [reply CMakeLists.txt:50]: Pelase try to compile to check if it's OK.

Comment (Shatur) [reply CMakeLists.txt:50]: I currently don't have access to my Windows machine, this is why I asked you to try it yourself first.

Comment (bovirus) [reply CMakeLists.txt:50]: Ok. Please help me to fix the ECM error installing ECM manually.

File: CMakeLists.txt (hunk 1 above)
Comment (bovirus) [reply CMakeLists.txt]: I aligned it for a better readibility

Comment (Shatur) [reply CMakeLists.txt]: I see, but I would refer to avoid alignment.

Comment (bovirus) [reply CMakeLists.txt]: As you like. 
For me alignment is better to check easiliy text strings.

**Pull request closed without merging**